src/Frontend/Page/Admin.hs

```diff
@@ -288,7 +288,9 @@ adminQuorum :: ActionM m => FormPageHandler m PageAdminSettingsQuorum
 adminQuorum =
     FormPageHandler
         (PageAdminSettingsQuorum <$> query (view dbQuorums))
-        (update . SaveQuorums)
+        (\qs -> do
+            update $ SaveQuorums qs
+            addMessage "Quorums are saved.")
```

Comment:
or:

```haskell
         (update . SaveQuorums >=> const (addMessage "Quorums are saved."))
```

(What would you name something that behaves to `>>` as `>=>` behaves to `>>=`?)

We can keep your version if you prefer.